src/session.ts

```diff
@@ -29,25 +31,96 @@ export interface SessionContext<S extends object> extends Context {
  * you can [install persistent session middleware from npm](https://www.npmjs.com/search?q=telegraf-session),
  * or pass custom `storage`.
  *
- * @example https://github.com/telegraf/telegraf/blob/develop/docs/examples/session-bot.ts
- * @deprecated https://github.com/telegraf/telegraf/issues/1372#issuecomment-782668499
+ * @example https://github.com/feathers-studio/telegraf-docs/blob/master/examples/session-bot.ts
  */
 export function session<S extends object>(
   options?: SessionOptions<S>
 ): MiddlewareFn<SessionContext<S>> {
   const getSessionKey = options?.getSessionKey ?? defaultGetSessionKey
   const store = options?.store ?? new MemorySessionStore()
```

Comment:
```suggestion
  const store = options?.store ?? new Map()
```

Reply:
Let's make this change in v5. This PR targets v4.